README.md

```diff
@@ -2,10 +2,12 @@
 
 ## Application Description
 
-TODO
+Generate multiple offline wallets for several popular cryptocurrencies.
```

Comment:
I think we should clarify the terminology around "wallet" and "address". The way that I've been thinking about it is that the "wallet" is the file that's generated with a specific mnemonic (and possibly passphrase). A wallet will contain one or more "addresses", which are specific to a cryptocurrency and BIP-44 path.

Unfortunately, I don't currently use the terms wallet and address that way in the code. I should refactor `Wallet` and the derived classes to do so.

This is all preamble to explain why I think that the text should instead say:
```
Generate offline wallets for several popular cryptocurrencies.
```

Reply:
Thanks for the clarification